docs/source/config/details.rst

```diff
@@ -228,3 +228,52 @@ a :ref:`startup file <startup_files>`::
 For more information on filters and what you can do with the ``event`` object,
 `see the prompt_toolkit docs
 <http://python-prompt-toolkit.readthedocs.io/en/latest/pages/building_prompts.html#adding-custom-key-bindings>`__.
+
+
+Enter to execute
+----------------
+
+As the exact behavior of :kbd:`Enter` is subject to disagreement, it has been
+special cased in order for users to completely configure the behavior they like.
+Hence you can have enter to start your coffee machine on even days of odd month,
```

Comment:
Let's put the funny example after the serious example, and maybe make it a bit shorter.